packages/login/src/modules/sagas.js

```diff
@@ -17,15 +17,30 @@ export const DEFAULT_TIMEOUT = 30
 export const textResourceSelector = (state, key) => state.intl.messages[key] || key
 export const loginSelector = state => state.login
 
-export function doLoginRequest(data) {
-  return fetch(`${__BACKEND_URL__}/nice2/login`, getOptions(data))
+export function doRequest(url, options) {
+  return new Promise(resolve => {
+    fetch(url, options)
+      .then(resp => {
+        resp.json().then(json => resolve(json))
+      })
+      .catch(e => {
+        if (console && console.error) {
```

Comment:
does console.error work in IE? I just know that is not standard: https://developer.mozilla.org/es/docs/Web/API/Console/error
And if console.error is undefined, the error message will never be shown